pyfr/solvers/base/elements.py

```diff
@@ -39,6 +39,10 @@ def __init__(self, basiscls, eles, cfg):
         # If we need quadrature points or not
         haveqpts = 'flux' in self.antialias
 
+        # If we are doing gradient fusion
+        self.grad_fusion = (cfg.getbool('backend', 'grad-fusion', True) and
```

Comment:
Should be part of the solver rather than the backend (this code is `pyfr/solvers...` rather than `pyfr/backends`)